integration_tests/test_str_01.py

```diff
@@ -37,6 +37,30 @@ def test_str_slice():
     # TODO:
     # assert a[0:5:-1] == ""
 
+def test_str_isalpha():
+    a:str = "helloworld"
+    b:str = "hj kl"
+    res: bool = a.isalpha()
+    res2: bool = b.isalpha()
+    assert res == True 
+    assert res2 == False
```

Comment:
Let's add a test for empty string.
Also, add "a12(){}A".

Reply:
I think you forgot to add `d: str = ""` and test `d.isalpha()`, `d.istitle()`.

Reply:
Added, Again I needed to make changes in `istitle` code. I am feeling confident with `istitle` and `title`, I will revisit them while porting it in CPP as we have to implement these in CPP as others `is*` are implemented in both cpp and Python.